Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.h

```diff
@@ -116,6 +116,10 @@ class MediaPlayerPrivateGStreamerMSE : public MediaPlayerPrivateGStreamer {
 
     bool m_isWaitingForPreroll = true;
     MediaPlayer::ReadyState m_mediaSourceReadyState = MediaPlayer::ReadyState::HaveNothing;
+
+    static unsigned mediaMaxWidth;
+    static unsigned mediaMaxHeight;
+    static unsigned mediaMaxFrameRate;
```

Comment:
Add another boolean static variable to flag if parsing has been attempted (even if it was unsuccessful).